run_db.sh

```diff
@@ -0,0 +1,3 @@
+
+echo "Starting catalina db..."
+docker run -d -p "$DB_HOST_PORT:3306" --name catalina_db "$DB_IMAGE_NAME:$DB_IMAGE_TAG"
```

Comment:
Realized myself that internal port is fixed to mysql. Internal is up to DB engine like mysql, mssql, postgres, etc

I will add a new param so it can be dynamic.